src/NUnitFramework/tests/Attributes/ValueSourceTests.cs

```diff
@@ -242,14 +243,15 @@ public void TestNameIntrospectsArrayValues()
             TestSuite suite = TestBuilder.MakeParameterizedMethodSuite(
                 GetType(), nameof(MethodWithArrayArguments));
 
-            Assert.That(suite.TestCaseCount, Is.EqualTo(4));
+            Assert.That(suite.TestCaseCount, Is.EqualTo(5));
 
             Assert.Multiple(() =>
             {
                 Assert.That(suite.Tests[0].Name, Is.EqualTo(@"MethodWithArrayArguments([1, ""text"", System.Object])"));
                 Assert.That(suite.Tests[1].Name, Is.EqualTo(@"MethodWithArrayArguments([])"));
                 Assert.That(suite.Tests[2].Name, Is.EqualTo(@"MethodWithArrayArguments([1, Int32[], 4])"));
                 Assert.That(suite.Tests[3].Name, Is.EqualTo(@"MethodWithArrayArguments([1, 2, 3, 4, 5, ...])"));
+                Assert.That(suite.Tests[4].Name, Is.EqualTo(@"MethodWithArrayArguments([System.Byte[,]])"));
```

Comment:
Last result

Reply:
That looks wrong to me since the arg type seems to read as "a single-dimensioned array containing a System.Byte[,]" rather than as above where it's just a System.Byte[,].

Reply:
@CharliePoole Isn't it, though? `new object[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 }` vs `new object[] { new byte[,] { { 1, 2 }, { 2, 3 } } }`.

Reply:
@CharliePoole Isn't it a single-dimensioned array containing a System.Byte[,], though?
`new object[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 }` vs
`new object[] { new byte[,] { { 1, 2 }, { 2, 3 } } }`.

Reply:
Isn't the new object[] simply the array of arts rather. Than one of the arts? That's what I assumed from your comments. I can't see the code  on my phone.